src/pyclvm/instance/stop.py

```diff
@@ -3,11 +3,16 @@
 
 
 def _stop_instance(instance_name: str, instance: Instance) -> None:
-    if instance.state.Name == "running":
+    _state = instance.state.Name
+    if _state == "stopped":
```

Comment:
How about to create map of states with corresponding actions?
```
{"stopped": `what to do`,
"terminated": `what to do`,
"running": `what to do`}
```

This will eliminate the if/elif statements.